docs/cli/graphql-transformer/directives.md

```diff
@@ -476,60 +476,66 @@ type Post
-- **update**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
-- **delete**: Add conditional update that checks the stored *ownerField* is the same as `$ctx.identity.username`.
+- **create**: Inject the logged in user's `$ctx.identity.username` as the `ownerField` automatically.
+- **update**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
+- **delete**: Add conditional update that checks the stored `ownerField` is the same as `$ctx.identity.username`.
 
-**Note**: When specifying operations as a part of the @auth rule, the operations not included in the list are not protected by default. For example, let's say you have the following schema:
+**Note**: When specifying operations as a part of the `@auth` rule, the operations not included in the list are not protected by default.
+
+Let's take a look at a few examples:
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [read] }]) {
+  @auth(rules: [{ allow: owner }]) {
   id: ID!
   updatedAt: AWSDateTime!
   content: String!
 }
 ```
 
-In this schema, only the owner of the object has the authorization to perform read (getTodo and listTodos) operations on the owner created object. But this does not prevent any other owner (any user other than the creator or owner of the object) to update/delete some other owner's object.
+In this schema, only the owner of the object has the authorization to perform read (`getTodo` and `listTodos`) and update operations (`updateTodo` and `deleteTodo`) on the owner created object. This prevents any other owner (any user other than the creator of the object) to update/delete some other owner's object.
+
 Here's a truth table for the above-mentioned schema. In the table below `other` refers to any user other than the creator or owner of the object.
 
 |       | getTodo | listTodos | createTodo | updateTodo | deleteTodo |
 |:------|:-------:|:---------:|:----------:|:----------:|:----------:|
 | owner |    ✅   |     ✅    |     ✅     |      ✅   |     ✅     |
-| other |    ❌   |     ❌    |     ✅     |      ✅   |     ✅     |
+| other |    ❌   |     ❌    |     ✅     |      ❌   |     ❌     |
 
-If you want to prevent updates and deletes operations, you would need to modify the @auth rule to explicitly include the `update` and `delete` operation and your schema should look like the following:
+Next, let's say that we wanted wanted to modify the schema to allow anyone to create, read, and update, but only the owner to be able to delete.
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [read, update, delete] }]) {
+  @auth(rules: [{ allow: owner, operations: [create, delete] }]) {
   id: ID!
   updatedAt: AWSDateTime!
   content: String!
 }
 ```
 
+In this schema, only the owner of the object has the authorization to perform delete operations on the owner created object, but anyone can read or update them.
```

Comment:
"This is because read and update aren't specified as owner-only actions, so all users are able to perform them. Since delete is specified as an owner only action, only the object's creator can delete the object." Something, something?

Reply:
🔥🔥🔥🔥